libs/gl-client/src/utils.rs

```diff
@@ -0,0 +1,31 @@
+use anyhow::{anyhow, Result};
+use crate::tls::TlsConfig;
+
+pub fn scheduler_uri() -> String {
+    std::env::var("GL_SCHEDULER_GRPC_URI")
+        .unwrap_or_else(|_| "https://scheduler.gl.blckstrm.com".to_string())
+}
+
+pub fn get_node_id_from_tls_config(tls_config: &TlsConfig) -> Result<Vec<u8>> {
```

Comment:
I moved this out to utils.rs because it was a public module. I considered putting this method in the TlsConfig, but with node_id not being directly linked to TLS certificates, I didn't think it made sense for it to be grouped in.